app/controllers/application_controller.rb

```diff
@@ -70,6 +71,8 @@ def skip_sentry_exception_types
 
     va_exception =
       case exception
+      when Pundit::NotAuthorizedError
+        Common::Exceptions::Forbidden.new(detail: 'User does not have access to the requested resource')
```

Comment:
This is nice. We currently have some more specific messaging for Secure Messaging, Health Records, and Rx. I will be curious to see how this would work with those products. I'm guessing the lower level controllers can just override if necessary.